src/snapshots/new_nu_parser__test__node_output@invalid_types.nu.snap

```diff
@@ -44,22 +45,22 @@ input_file: tests/invalid_types.nu
 4: int
 5: unknown
 6: string
-7: unknown
+7: ()
 8: list<unknown>
-9: unknown
-10: unknown
-11: unknown
-12: ()
+9: list<unknown>
```

Comment:
I think Param should also be `forbidden` since it won't be directly evaluated?

Reply:
I was thinking we can do some better typechecking by checking each param and if it is type compatible to the argument of the function once we have the facilities for that.